Monika After Story/game/styles.rpy

```diff
@@ -68,15 +69,20 @@ init -200 python in mas_ui:
     #    TALL for menus w/o final buttons
     #    MEDIUM for menus w/ one final button
     #    LOW for menus w/ 2 final buttons
+    #    VLOW for menus w/ 3 final buttons
 
     SCROLLABLE_MENU_TALL_AREA = (SCROLLABLE_MENU_X, SCROLLABLE_MENU_Y, SCROLLABLE_MENU_W, SCROLLABLE_MENU_TALL_H)
     SCROLLABLE_MENU_MEDIUM_AREA = (SCROLLABLE_MENU_X, SCROLLABLE_MENU_Y, SCROLLABLE_MENU_W, SCROLLABLE_MENU_MEDIUM_H)
     SCROLLABLE_MENU_LOW_AREA = (SCROLLABLE_MENU_X, SCROLLABLE_MENU_Y, SCROLLABLE_MENU_W, SCROLLABLE_MENU_LOW_H)
+    SCROLLABLE_MENU_VLOW_AREA = (SCROLLABLE_MENU_X, SCROLLABLE_MENU_Y, SCROLLABLE_MENU_W, SCROLLABLE_MENU_VLOW_H)
 
     SCROLLABLE_MENU_TXT_TALL_AREA = (SCROLLABLE_MENU_X, SCROLLABLE_MENU_Y, SCROLLABLE_MENU_W, SCROLLABLE_MENU_TXT_TALL_H)
     SCROLLABLE_MENU_TXT_MEDIUM_AREA = (SCROLLABLE_MENU_X, SCROLLABLE_MENU_Y, SCROLLABLE_MENU_W, SCROLLABLE_MENU_TXT_MEDIUM_H)
     SCROLLABLE_MENU_TXT_LOW_AREA = (SCROLLABLE_MENU_X, SCROLLABLE_MENU_Y, SCROLLABLE_MENU_W, SCROLLABLE_MENU_TXT_LOW_H)
 
+    #Sort function for scrollable menu prompts
+    SCROLLABLE_MENU_PROMPT_SORT_KEY = key=lambda x: x[0]
```

Comment:
did you really need a sort key for this? by default, tuples are sorted in left-right order, so the tuples will be sorted based on their first index without this.